jobs/transforms/user_validation_test.py

```diff
@@ -231,15 +229,6 @@ def test_model_with_valid_draft_change_list_last_updated(self):
         self.assert_pcoll_equal(output, [])
 
 
-class RelationshipsOfTests(test_utils.TestBase):
```

Comment:
Why are we deleting these tests?

Reply:
Sorry I thought recently added changes made sure the decorators were working correctly and thus no more need of this tests?
I have added them again PTAL cc @rohitkatlaa

Reply:
I thought the additional tests(added in this PR) were not necessary because they were checking the same right?

Reply:
Ah I see, that's my bad. I had a change of heart with this because I had to debug a situation where the relationships didn't work in Python 3. Since then, I think it's important after all to keep these tests.

Reply:
Ok then in that case, I am sorry for the confusion @YBCS please add the necessary tests for all the relations. Thanks!